packages/contracts-bedrock/contracts/L2/GasPriceOracle.sol

```diff
@@ -164,7 +156,7 @@ contract GasPriceOracle is Ownable, Semver {
                 total += 16;
             }
         }
-        uint256 unsigned = total + overhead;
+        uint256 unsigned = total + overhead();
```

Comment:
This adds a fair bit of gas overhead, but it's only meant to be called off-chain right?

Reply:
Yes its only meant to be called offchain